packages/core/src/bag3d/core/assets/ahn/metadata.py

```diff
@@ -247,7 +247,6 @@ def compute_load_metadata(
             ST_SetSRID(ST_GeomFromGeoJSON({boundary}), 28992)
         );    
         """).format(**query_params)
-    context.log.info(conn.print_query(query))
     conn.send_query(query)
```

Comment:
It handy if we log the sql queries, at least in debug level. It make debugging much more easy if we can look up the logs for the exact sql query that was used. 
Please put the query logging back with debug level.